example/build.gradle

```diff
@@ -13,8 +13,8 @@ android {
         applicationId "com.processout.example"
         minSdk rootProject.ext.minSdkVersion
         targetSdk rootProject.ext.targetSdkVersion
-        versionCode file('../version.resolved').getText().replace(".", "").toInteger()
-        versionName file('../version.resolved').getText()
+        versionCode rootProject.ext.publishVersion.replace(".", "").toInteger()
```

Comment:
Good point, it would matter if we where distributing example app on Play Store or elsewhere, currently not critical.
We can adopt UpdateVersion.sh script for such cases.

Reply:
Android Library doesn't have `versionCode` so it doesn't affect our SDKs.